Iterate.py

```diff
@@ -118,23 +118,40 @@ def iterate(si, st):
 
     if si.verbosity>2:
         print(f"qpllu0: {si.qpllu0:.3E} | nu: {st.nu:.3E} | Tu: {st.Tu:.1f} | cz: {st.cz:.3E} | cvar: {st.cvar:.2E}", end = "")
-
-    result = odeint(LengFunc, 
+    
+    # result = odeint(LengFunc, 
+    #                 y0 = [st.qpllt/si.B(st.s[0]),si.Tt],
+    #                 t = st.s,
+    #                 args = (si, st)
+    #                 )
+    
+    result = solve_ivp(LengFunc, 
+                    t_span = (st.s[0], st.s[-1]),

```

Comment:
Successfully implemented the solve_ivp solver. It's much more stable and more flexible than odeint and runs faster.